jwst/assign_wcs/nirspec.py

```diff
@@ -554,7 +554,7 @@ def get_open_slits(input_model, reference_files=None, slit_y_range=(-0.55, 0.55)
         slits = validate_open_slits(input_model, slits, reference_files)
         log.info(
             f"Slits projected on detector {input_model.meta.instrument.detector}: "
-            f"{[sl.name for sl in slits]}"
+            f"{[int(sl.name) for sl in slits]}"
```

Comment:
Do you know if this works when there are fixed slits in the list of open slits?  I seem to recall that they had their sl.name set to the FS string, which I think would throw and error here if they are printed in this same statement.

Reply:
good question will check

Reply:
sure enough this does not work right